src/partials/modal.html

```diff
@@ -0,0 +1,51 @@
+<div class="backdrop is-hidden" data-modal>
+  <div class="modal">
+    <div class="modal-top">
+      <button
+        type="button"
+        class="header-btn-burger header-icon-cross"
+        aria-expanded="false"
+        data-modal-close
+      >
+        <svg
+          class="header-icon-cross"
+          width="20"
+          height="20"
+          aria-label="Переклчатель мобильного меню"
+        >
+          <use
+            class="header-icon-cross"
+            xlink:href="./images/sprite-modal-btn.svg#cross"
```

Comment:
xlink не нужно использовать, так как проект не запускается

```suggestion
            href="./images/sprite-modal-btn.svg#cross"
```